c/aws-c-common/Makefile

```diff
@@ -0,0 +1,29 @@
+LEVEL := ../
+
+CC.Arch := 64
+
+COMMON_WARNINGS := \
+	-Wno-error=shift-count-overflow \
+	-Wno-error=overflow \
+	-Wno-error=implicit-function-declaration \
+	-Wno-error=incompatible-pointer-types \
+	-Wno-error=int-conversion \
+	-Wno-error=address \
+
+GCC_WARNINGS += \
+	-Wno-error=format= \
+	-Wno-error=discarded-qualifiers
+#	-Wno-error=builtin-declaration-mismatch
+
+CLANG_WARNINGS += \
+	-Wno-error=uninitialized
```

Comment:
This makefile excludes several warnings about undefined behavior. SV-COMP tasks need to be free of undefined behavior (except if the task has the goal of checking for that behavior). Are you sure that these kinds of behavior do not actually occur in practice and the compilers only produce false alarms?

Reply:
I could potentially fix these issues in my fork of the original sources, not sure though how much effort that would be. As you mentioned there are many such warnings and tracking them down might take a while...